### PR TITLE
add vimeo.com into oEmbed $provider_name

### PR DESCRIPTION
**Pull request by abdusfauzi**
Fix vimeo.com oEmbed figure wrapping.

### CODE REVIEW

**Comment by kkranzo** [comment]
When will just a basic iframe (wrapping any standard iframe/non oEmbed tag in the content) wrap be added?

**Comment by diegoquinteiro** [comment]
@kkranzo as of the latest version you can already use `<div class="embed">...</div>` as a generic wrapper. Any iframe should also work too (let me know if it does not).

File: embeds.php [hunk 1]
```diff
@@ -41,8 +41,12 @@ function instant_articles_embed_oembed_html( $html, $url, $attr, $post_ID ) {
 		$provider_name = 'twitter';
 	} elseif( false !== strpos( $providerURL, 'youtube.com' ) ) {
 		$provider_name = 'youtube';
+    } elseif( false !== strpos( $providerURL, 'vimeo.com' ) ) {
```
Comment (diegoquinteiro) [inline embeds.php]: Please fix indentation to use tabs.

**Comment by diegoquinteiro** [comment]
Indentation looks good, but still need to resolve the conflicts

**Pull request merged**